docs/specification/current.md

```diff
@@ -1095,6 +1095,76 @@ As each hook progresses through a process of being defined, tested, implemented,
+In addition to the attributes specified in the [Discovery](#discovery) section of this page, a new `extension` resource will be required.
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | ---------
+`extension` | REQUIRED | *object* | 
+ 
+ 
+```
+ {
+  "services": [
+    {
+      "hook": "order-select",
+      "title": "Static CDS Service Example",
+      "extension": {
+        "configuration-items": [
+          {
+            "code": <an identifier for the configuration option>,
+            "type": <data type of the configuration option, e.g. boolean>,
+            "name": <human readable name of the configuration option>,
+            "description": <human readable description of the configuration option>
+          },
+
+          ... other configuration options ...
+        ]
+    },
+    {
+      "hook": "order-sign",
+      "title": "Static CDS Service Example",
+      "extension": {
+        "configuration-items": [
+          {
+            "code": <an identifier for the configuration option>,
+            "type": <data type of the configuration option, e.g. boolean>,
+            "name": <human readable name of the configuration option>,
+            "description": <human readable description of the configuration option>
+          },
+
+          ... other configuration options ...
+        ]
+      }, ...
+
+snipped for brevity
+```
+
+##### Extension
+
+###### Configuration Items
+
+The CDS Service MUST process configuration options sent as within an extension resource with the CDS Hooks request in the field `configuration-items`. There are two standart configuration options:  `cache-for-order-sign-filtering` (only used in `order-select`), `filter-out-repeated-alerts` (only used in `order-sign`). All of these configuration options accept Boolean values and are required. The `cache-for-order-sign-filtering` option is used to cache the medication resource during `order-select` and is used as a reference during `order-sign` if `filter-out-repeated-alerts` is set to TRUE. The `filter-out-repeated-alerts` option is used to hide card results if they were shown in the `order-select` results.
+
+#### FHIR Server Request
+
+With coordination, the parse and pre-process event for an `order-select` request is nearly identical to what occurs with an `order-select` request without coordination except that the CDS Service checks to see if `cache-for-order-sign-filtering` was set to TRUE and, if so, it caches information that it will later use to filter duplicate CDS Hooks cards from a future `order-sign` request. Then, during the parse and pre-process phase for `order-sign`, the CDS service checks to see if `filter-out-repeated-alerts` is also set to TRUE. If so, it then checks the cached data for a match on specific resources and information to identify if the `order-sign` card response would be a duplication. If so, repeated alert cards will not be returned. 
```

Comment:
Because the base spec is a general framework for communicating between CDS Clients and Services, we should be careful not to add content that is too hook-specific. True, there may be examples of hooks, but attributes such as "cache-for-order-sign-filtering" are specific to ordering hooks and would not be present in the base spec.